packages/date-time-utilities/src/dateGrid/DateGrid.test.ts

```diff
@@ -0,0 +1,717 @@
+import { DayOfWeek, DateRangeType, FirstWeekOfYear } from '../dateValues/DateValues';
+import * as DateGrid from './DateGrid';
+import { IDay, IDayGridOptions, IRestrictedDatesOptions, IAvailableDateOptions, IGridStrings } from './DateGrid.types';
+
+enum Months {
+  Jan = 0,
+  Feb = 1,
+  Mar = 2,
+  Apr = 3,
+  May = 4,
+  Jun = 5,
+  Jul = 6,
+  Aug = 7,
+  Sep = 8,
+  Oct = 9,
+  Nov = 10,
+  Dec = 11,
+}
+
+describe('DateUtils', () => {
+  const strings: IGridStrings = {
+    days: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'],
+    months: [
+      'January',
+      'February',
+      'March',
+      'April',
+      'May',
+      'June',
+      'July',
+      'August',
+      'September',
+      'October',
+      'November',
+      'December',
+    ],
+    shortDays: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'],
+    shortMonths: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+  };
+
+  describe('getDayGrid', () => {
+    const defaultOptions: IDayGridOptions = {
+      selectedDate: new Date(2016, Months.Apr, 1),
+      navigatedDate: new Date(2016, Months.Apr, 1),
+      firstDayOfWeek: DayOfWeek.Sunday,
+      firstWeekOfYear: FirstWeekOfYear.FirstDay,
+      dateRangeType: DateRangeType.Day,
+    };
+
+    const defautlResult: IDay[][] = [
```

Comment:
It seems like this could be somewhat simplified by just taking a jest snapshot of the output (and validating it's what you typed here.)

Reply:
Great idea, thank you :)